src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -243,3 +243,6 @@ nginx_sites:
        balancer: "{{ deploy_env }}_cas_formplayer/"
        proxy_next_upstream_tries: 1
        proxy_read_timeout: 900s
+     - name: /sentry
+       proxy_pass: "http://{{ groups['sentry'][0] }}:8080"
+       proxy_read_timeout: 900s
```

Comment:
This seems to be going to django and throwing 404

Reply:
I played around with this a bit but because Sentry uses redirects it won't work without making sentry aware that it's running with a prefix and I'm not sure that's possible. So I think a subdomain is likely the best approach. We should then do the same for graphana etc.

Reply:
I got this gist while looking around and tried but it gets stuck with redirect loop . https://gist.github.com/beeftornado/945d5180ffd402f5cd8d

Reply:
hmm, seems like it should be possible:
*nginx*
```
location /sentry {
		proxy_pass http://0.0.0.0:9000;
		proxy_redirect	off;

		proxy_set_header	Host	$host;
		proxy_set_header	SCRIPT_NAME	/sentry;
		proxy_set_header	X-Real-IP	$remote_addr;
		proxy_set_header	X-Forwarded-For	$proxy_add_x_forwarded_for;
		proxy_set_header	X-Forwarded-Proto	$scheme;
	}
```

*sentry config*
```
SENTRY_URL_PREFIX = 'http://<TOP_LEVEL_DOMAIN>.com/sentry'  # No trailing slash!
FORCE_SCRIPT_NAME = '/sentry'

SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
USE_X_FORWARDED_HOST = True

SENTRY_WEB_OPTIONS = {
    ....
    'secure_scheme_headers': {'X-FORWARDED-PROTO': 'https'},
}
```

Reply:
Yeah i tried this but couldn't figured out how to fix redirect loop.

Reply:
I played around with this for quite a while but couldn't fix the redirect loop

Reply:
Going with subdomain route.